doc/tor.1.txt

```diff
@@ -2605,6 +2605,15 @@ Relays publish most statistics in a document called the
 extra-info document. The following options affect the different
 types of statistics that Tor relays collect and publish:
 
+[[BandwidthStatistics]] **BandwidthStatistics** **0**|**1**::
+    Relays and bridges only.
+    When this option is enabled, Tor collects statistics about bandwidth
+    used each 24 hours for the last 5 days (if available). Reports
+    bandwidth statistics for OR and directory reads and writes,
+    and writes them to disk every 24 hours. Enables relay and bridge operators
+    to monitor bandwidth used by Tor. If ExtraInfoStatistics is enabled,
+    it will be published as part of the extra-info document. (Default: 1)
```

Comment:
Please describe what "auto" does. Check the rest of the manual page for examples.